applications/nrf5340_audio/dfu/conf/Kconfig.dfu

```diff
@@ -41,7 +32,8 @@ config BOOTLOADER_MCUBOOT
 	bool
 	default y
 
-config NCS_INCLUDE_RPMSG_CHILD_IMAGE
+
+config EMPTY_NET_CORE_CHILD_IMAGE
```

Comment:
Could you elaborate on this?

Reply:
This is dummy netcore appliation for generating b0n.
We used hci_rpmsg as dummy app before.
Since we use prebuilt controller, it makes no sense to build hci_rpmsg.
So I decide to use truly dummy one.

Reply:
Thanks. Can we rename or add a comment that this is a dummy?

Reply:
Add

> 	bool "Dummy Net core application"